net.retrodeck.retrodeck.yml

```diff
@@ -425,8 +425,8 @@ modules:
       - es-core
     sources:
       - type: git
-        url: https://github.com/XargonWan/RetroDECK-ES-DE
-        branch: retrodeck-main
+        url: https://github.com/monkeyx-net/RetroDECK-ES-DE-UK.git
```

Comment:
Same as above.

Reply:
I wanted to draw attention to how I got it to build from my repo.

Without RDREPO it would not build. Without manually changing repos here

Need to document what this is better for when it switches back to the main repo or another repo

Reply:
Same as above again :P